### ISSUE TITLE
Snow Forming on bridges

### ISSUE THREAD

**Issue by sketchmacaw** [sketchmacaw/MacawsModsIssues#27]
Snow is forming on bridges

**Comment by AechtRob** [comment]
Just to add a bit more.... snow forms floating above the bridge, making it impossible to walk over the bridge unless you remove the snow first, as the snow is lying at face-level.

**Comment by AechtRob** [comment]
I'd place bets that you need this override adding to your bridge block class:

		```
@Override
		public net.minecraft.block.state.BlockFaceShape getBlockFaceShape(net.minecraft.world.IBlockAccess worldIn, net.minecraft.block.state.IBlockState state, net.minecraft.util.math.BlockPos pos, net.minecraft.util.EnumFacing face)
	    {
	        if (face == net.minecraft.util.EnumFacing.UP) {
				return net.minecraft.block.state.BlockFaceShape.UNDEFINED;
			}
			
			return super.getBlockFaceShape(worldIn, state, pos, face);
	    }
```

**Comment by AdamNisbett** [comment]
Though perhaps not as impactful, snow on the roofs is also similarly problematic in appearance as it does not follow the slope of the roof, but rather floats as a horizontal layer above the roof block space.

**Comment by AechtRob** [comment]
The same fix applied to the base roof block class would fix it. If he picks it up and updates......

**Comment by sketchmacaw** [comment]
February

**Comment by jmessy919** [comment]
Hi all - just checking if the snow issue on the roofs got fixed - no rush, just curious if I need to update or anything like that. Thank you - this mod is amazing and my daughter loves it!

**Comment by sketchmacaw** [comment]
Fixed in the upcoming 1.0.6.
